library/styleguideComponents/ReactComponent/Utils/experienceExampleCard.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Card from '../../../../src/components/Card';
+import githubLogo from './image/github.svg';
+import './styles.css';
+
+export default function ExperienceExampleCard({ results }) {
+    return results.map(({ data, id }) => (
+        <div>
+            <div className="react-rainbow-utils-text">
+                <h1 className="react-rainbow-utils-text_header">Real experience examples</h1>
+                <p className="react-rainbow-utils-text-description">
+                    The Real experience examples will help you get
+                    started building your app using the Button Component.
```

Comment:
using the Button Component?
here you need to use:
````using the ${componentName} Component ...````
but here this is not correctly because you are rendering this many times as the results length